BUILD.md

```diff
@@ -76,15 +86,16 @@ maturin develop --release
 From the repository root:
 ```bash
 cd python/vegafusion-jupyter/
-cd npm link vegafusion-wasm
+npm link vegafusion-wasm
+npm install
 pip install -e ".[test]"
```

Comment:
I believe something happens during this step that runs webpack or does dome FE build process - most of the new `npm install` lines above I had to add in order to get this command to succeed